ui/apps/platform/src/Containers/Administration/Events/AdministrationEventPage.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { ReactElement, useEffect, useState } from 'react';
+import {
+    Alert,
+    Breadcrumb,
+    BreadcrumbItem,
+    Bullseye,
+    Flex,
+    PageSection,
+    Spinner,
+    Title,
+} from '@patternfly/react-core';
+
+import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
+import PageTitle from 'Components/PageTitle';
+import { AdministrationEvent, getAdministrationEvent } from 'services/AdministrationEventsService';
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+import { administrationEventsBasePath } from 'routePaths';
+
+import { getLevelText } from './AdministrationEvent';
+import AdministrationEventDescription from './AdministrationEventDescription';
+
+export type AdministrationEventPageProps = {
+    id: string;
+};
+
+function AdministrationEventPage({ id }: AdministrationEventPageProps): ReactElement {
+    const [isLoading, setIsLoading] = useState(false);
+    const [event, setEvent] = useState<AdministrationEvent | null>(null);
+    const [errorMessage, setErrorMessage] = useState('');
+
+    useEffect(() => {
+        setIsLoading(true);
+        getAdministrationEvent(id)
+            .then((eventArg) => {
+                setEvent(eventArg);
+                setErrorMessage('');
+            })
+            .catch((error) => {
+                setEvent(null);
+                setErrorMessage(getAxiosErrorMessage(error));
+            })
+            .finally(() => {
+                setIsLoading(false);
+            });
+    }, [id]);
+
+    const h1 = event ? `${getLevelText(event.level)} - ${event.domain}` : 'Administration event';
+
+    /* eslint-disable no-nested-ternary */
```

Comment:
If this is only a stylistic eslint rule, and we chose to disable it in some cases, can we just disable it globally?

Reply:
Indeed, let’s discuss in team meeting. After working with descriptive linguists, case analysis from specific to generic is deeply ingrained and nested ternary fits that pattern. Although some people loath the consistent alignment that prettier now enforces, that formatting makes it even clearer to me.